src/components/home/modals/AddToken.vue

```diff
@@ -101,7 +100,9 @@ export default {
             this.$http.get(url).then(response => {
```

Comment:
This bug is caused by empty this.tokenId. So I suggest this bug fix by stopping http request if this.tokenId is empty. Like this:
```
if (!this.tokenId) {
   return;
}
```